core/templates/pages/exploration-editor-page/editor-tab/state-version-history/state-version-history.component.ts

```diff
@@ -41,6 +41,15 @@ export class StateVersionHistoryComponent {
     );
   }
 
+  // The return value of the below function can never be null because it is
+  // called only when canShowExploreVersionHistoryButton() returns true.
```

Comment:
In this case, the correct thing to do is to throw an error if it ends up being null and make the type stricter instead, so that you get a clear signal if it's misbehaving.

Reply:
Done.